nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/queue/ui/queue-listing/flowfile-table/flowfile-table.component.html

```diff
@@ -112,23 +112,17 @@ <h3 class="queue-listing-header primary-color">{{ selectedConnection?.label }}</
                     <th mat-header-cell *matHeaderCellDef></th>
                     <td mat-cell *matCellDef="let item">
                         <div class="flex items-center justify-end gap-x-2">
-                            @if (
-                                item.size > 0 ||
-                                (contentViewerAvailable && item.size > 0) ||
-                                currentUser.provenancePermissions.canRead
-                            ) {
-                                <button
-                                    mat-icon-button
-                                    type="button"
-                                    [matMenuTriggerFor]="actionMenu"
-                                    class="h-16 w-16 flex items-center justify-center icon global-menu">
-                                    <i class="fa fa-ellipsis-v"></i>
-                                </button>
-                            }
+                            <button
+                                mat-icon-button
+                                type="button"
+                                [matMenuTriggerFor]="actionMenu"
+                                class="h-16 w-16 flex items-center justify-center icon global-menu">
+                                <i class="fa fa-ellipsis-v"></i>
+                            </button>
                             <mat-menu #actionMenu="matMenu" xPosition="before">
                                 <button mat-menu-item (click)="viewFlowFileClicked(item)">
                                     <i class="fa fa-info-circle primary-color mr-2"></i>
-                                    View Details
+                                    View details
```

Comment:
I assume there is no reason to change the case
```suggestion
                                    View Details
```

Reply:
Yes. It is consistent with the case of other options.